### DISCUSSION TITLE
Is it possible to login ?

### DISCUSSION THREAD

**Discussion by SirLouen** [pear-devs/pear-desktop#592] [Q&A] [answered]
Can't find any info about this but when I try to login I observe that the browser is not supported (browser is not supported)

There is a plugin to remove login so I'm suspicious that this is not simply supported, although, it would be completely awesome to make it possible (specially to access my playlists)

Any information on this topic?

**Comment by Araxeus** [comment, marked as answer]
Did you try https://github.com/th-ch/youtube-music/releases/tag/v1.15.0?

**Reply by SirLouen** [reply]
Great, it solves this issue.

I downloaded firstly the stable version.

**Reply by Araxeus** [reply]
@th-ch need new stable version 🙂

**Comment by inson1** [comment]
@SirLouen May you close this if it is solved? :)
